config/locales/en.yml

```diff
@@ -273,10 +285,10 @@ en:
           info_html: |
             <p class="govuk-body">For example, you should tell everyone who:</p>
             <ul class="govuk-list govuk-list--bullet">
-              <li>cares for the children (but is not their parent), including social services if the children are in local authority accommodation (such as foster care or a children's home)</li>
-              <li>is currently involved in another court case or named in a current court order that concerns the children and is relevant to this application</li>
-              <li>the child has lived with for at least 3 years prior to this application</li>
-            </ul>
+                               <li>cares for the children (but is not their parent), including social services if the children are in local authority accommodation (such as foster care or a children's home)</li>
```

Comment:
Some weird indentation here, probably by mistake 😄

Reply:
oh yeah weird! Will fix that up